protos/flyteidl/plugins/sagemaker/hyperparameter_tuning_job.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+package flyteidl.plugins.sagemaker;
+
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/plugins";
+
+import "flyteidl/plugins/sagemaker/parameter_ranges.proto";
+import "flyteidl/plugins/sagemaker/training_job.proto";
+
+// The hyperparameter tuning job
```

Comment:
You might want to add references to AWS Reference Docs about the parameters throughout the doc... and explain that these are merely passthrough (or case they are not, explain how we are using them)...